query-builder/src/main/java/com/datastax/oss/driver/api/querybuilder/insert/Insert.java

```diff
@@ -45,9 +45,23 @@
   @NonNull
   Insert usingTimestamp(@Nullable BindMarker bindMarker);
 
+  /**
+   * Adds a USING TTL clause to this statement with a literal value. Setting a value of {@code 0}
+   * will clear a previously set TTL.
```

Comment:
I find this a bit ambiguous because it's not clear whether this applies to the query builder or Cassandra.
* passing `null` removes the `USING TTL` clause from the generated query
* passing 0 will adds `USING TTL 0` clause to the generated query, which when executed will delete the TTL for the column(s)

Reply:
Good catch, I'll update the docs to reflect it.